packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -961,7 +947,7 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
   _onKeyDown(event) {
     super._onKeyDown(event);
 
-    const chips = Array.from(this._chips).slice(1);
+    const chips = this._chips;
```

Comment:
No need to call `.slice(1)` anymore because `this._chips` no longer includes overflow chip.